### DISCUSSION TITLE
How To Handle OOM Errors in Chrome?

### DISCUSSION THREAD

**Discussion by radiantone** [gpuweb/gpuweb#4735] [Q&A] [answered]
Is there a heap or memory limit setting somewhere in chrome to avoid this?

```
"vkAllocateMemory failed with VK_ERROR_OUT_OF_DEVICE_MEMORY\n    at CheckVkOOMThenSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:114)\n"
```

**Comment by greggman** [comment, marked as answer]
This is the spec repo/forum, not Chrome tech support. Maybe someone else can suggest a better place for this question.

**Reply by greggman** [reply]
maybe https://matrix.to/#/#webgpu-dawn:matrix.org

**Reply by kainino0x** [reply]
FYI I'm updating the discussion categories so that it'll be a bit clearer where to go for questions like this.
